lib/file-view.coffee

```diff
@@ -19,7 +19,11 @@ class FileView extends HTMLElement
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
 
-    @fileName.classList.add(FileIcons.getService().iconClassForPath(@file.path))
+    iconClass = FileIcons.getService().iconClassForPath(@file.path, this)
+    if iconClass
+      unless Array.isArray iconClass
```

Comment:
Are you referring to `@fileName.classList.add(iconClass…)`? I think the trailing `…` will work fine, adding `first` and `second` appropriately, without commas. As for when `iconClass` is a string with spaces, the code I proposed will throw an exception like the following:

```
Uncaught DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('foo bar') contains HTML space characters, which are not valid in tokens.
```

But I think this was the case also before this pull-request, and it seems nice to let `classList.add` enforce its default preconditions. Thoughts?